db/files.go

```diff
@@ -11,8 +11,8 @@ import (
 
 // AddFile is used to add the md5 of a file name which is uploaded to our application
 // this will enable us to randomize the URL without worrying about the file names
-func AddFile(fileName, token string) error {
-	err := taskQuery("insert into files values(?,?)", fileName, token)
+func AddFile(fileName, token, username string) error {
+	err := taskQuery("insert into files values(?,?,?,datetime())", fileName, token, username)
```

Comment:
```
userID, err := GetUserID(username)
	if err != nil {
		return nil
	}
err := taskQuery("insert into files values(?,?,?,datetime())", fileName, token, userID)
```

For your files table, we have a referential integrity with the user table and we fetch the userID from a function in database called GetUserID. This is what we pass in the taskQuery function otherwise it'll throw an error :-)

If you want to learn Go and contribute more, I am open to get PRs. Keep contributing!